### ISSUE TITLE
[PAGE] About 311 Data

### ISSUE THREAD

**Issue by mattyweb** [hackforla/311-data#960]
### Overview
Update the About us page to no longer be the landing page and will fit with the look and feel of the tool.

### Action Items / Open Questions
- [x] Is this the same content as v1? - NO
- [x] Is this the same design as v1? - NO
- [x] Define Changes
   - [ ] Add History
   - [x] Change look and feel to the new format
   - [x] Don't have giant image with arrow at the bottom.
   - [x] Under "How It Works" 
     - [x] Re-organize the steps to include "City  publishes data"
     - [x] Specify who owns each step
       - [x] A service request is reported to the City of LA
       - [x] The City distributes request to appropriate department for action
       - [x] City publishes data service request data
       - [x] 311-Data.org provides visualizations and analytics from the data 
       - [x] Users gain awareness and insights through search and visuals 

### Resources
https://311-data.org/about

### The prior page features and where are they now
- Overview of what 311 is: Each day, Los Angelenos report thousands of 311 requests all across LA to resolve issues such as illegal dumping and graffiti in their neighborhoods. These requests are then received by relevant agencies, such as the Police, Building and Safety, or Department of Transportation. The agency responds to the request, addresses it, and then closes it once it is fixed.
- [Open Data Initiative](https://data.lacity.org/)
- [Neighborhood Councils](https://empowerla.org/councils/)
- [Hack For LA](https://www.hackforla.org/)
- [EmpowerLA](https://empowerla.org/)
- [311 Data project](https://www.hackforla.org/projects/311-data)
- [Report Tool Alpha](https://hackforla.github.io/311-report/)
- logos of 
   - empowerla.org
   - hackforla.org
   - codeforamerica

- [Mockups in Figma - WIP](https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=2775%3A7702)

**Comment by danarchen** [comment]
We will update v2 to include contact us in NAV and any other necessary links.

**Comment by simrandhaliw** [comment]
Added the initial design idea to Figma (design studio section). Will discuss the idea with other designers and add it for review next week.

**Comment by adamkendis** [comment]
Status: New mockups https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=2775%3A7702

**Comment by pras4u** [comment]
@adamkendis  per @simrandhaliw this ticket is ready for Dev

**Comment by EchoProject** [comment]
@pras4u Do you remember what happened with this issue? The above says it was ready for dev, but we're unsure which of the About pages is the final version.

**Comment by ExperimentsInHonesty** [comment]
@EchoProject will re write using the image on the far right (white background) and add in another step (city publishes the data), and make it clear who owns each step

1. You the citizen inputs into city
2. City distributes to departments for action
3. city published data
4. our website uses the data the city publishes
5. you use our site to understand the nuances of the data

**Comment by ExperimentsInHonesty** [comment]
@EchoProject some of the text that was linked, is also not linked in the mockups (Neighborhood Councils) and some of the text that was there before is no longer there (e.g., Open Data Initiative, [Report Tool Alpha](https://hackforla.github.io/311-report/, etc).  So we are still going to need an audit of what is there and what is missing and make a decision about if we want to re add it, or it can be left off, or added someplace else.

**Comment by EchoProject** [comment]
I've updated the requirements to include: 
   - [ ] Under "How It Works" 
     - [ ] Re-organize the steps to include "City  publishes data"
     - [ ] Specify who owns each step
       - [ ] A service request is reported to the City of LA
       - [ ] The City distributes request to appropriate department for action
       - [ ] City publishes data service request data
       - [ ] 311-Data.org provides visualizations and analytics from the data 
       - [ ] Users gain awareness and insights through search and visuals 

I'm assigning this to @melissahermes99 to make the design changes. Melissa, please see Bonnie's comment above for including links to the appropriate Figma panel. Please reach out with questions.

**Comment by melissahermes99** [comment]
The updated About Us page is located on [Version 2_Design Studie](https://www.figma.com/file/eF63O8yIGz2soQ4JJFfyoK/311_Data_UXUI?node-id=2775%3A7702) Under Sim's name to the far right. This is the updated white variant. Dhwani made a comment regarding the decision to exclude imagery on the page. I am not sure if that has been resolved yet. She also made note that the header was not consistent with the Design System. I have updated it to the correct header. I think there might've been confusion because the header from the Design System does not contain a "About Us" tab.
